cpp/src/rolling/rolling_detail.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Comment:
Could you change this header to use `#pragma once` instead of the `#ifndef ROLLING_DETAIL_HPP` pattern?

Reply:
Hah, previously my IDE always automatically generated such header guard. I had to explicitly stop it.